dashboard/src/pages/TreeDetails/Tabs/Tests/TestsTable.tsx

```diff
@@ -242,6 +224,24 @@ export function TestsTable({ testHistory }: ITestsTable): JSX.Element {
       return count;
     }, [rawData]);
 
+  const onClickFilter = useCallback(
```

Comment:
Any reason to move this block of code?

Reply:
Makes the order of the functions/variables consistent with the other tables. rawData - data - filterCount - onClickFilter - filters - table - headers - rows - return